From 49b506a79a319efa336cfc08adad1fffe6d874fa Mon Sep 17 00:00:00 2001
From: Ariel Aviad <aaviad@habana.ai>
Date: Sun, 04 May 2025 06:41:58 +0000
Subject: [PATCH] Revert "[SW-0] hl-smi: info: rm check before get fw fit ver"

This reverts commit 87d64613a9bce85df0f844bfec8c15f00ff73a66.
We are reverting this because, in non-Linux environments,
as gaudi3 we prefer not to print any information
and not to print N/A because the FIT version doesn't exists.

Change-Id: I2bb27a61c35b95168767eb47efde55eb297f7954
Signed-off-by: Ariel Aviad <aaviad@habana.ai>
---

diff --git a/app/hl-smi/info.cpp b/app/hl-smi/info.cpp
index d3140db..5a9c00d 100644
--- a/app/hl-smi/info.cpp
+++ b/app/hl-smi/info.cpp
@@ -424,11 +424,13 @@
 	HL_SMI_LOG_INFO("\tAIP UUID\t\t\t: {}", info_field);
 	hl_smi_aip_log_status(aip);
 
-	ret = hlml_get_firmware_fit_version(aip, fw_fit_version, sizeof(fw_fit_version));
-	if (ret == HLML_SUCCESS)
-		HL_SMI_LOG_INFO("\tFirmware [FIT] Version\t\t: {}", fw_fit_version);
-	else
-		HL_SMI_LOG_ERR("FAIL in function: hlml_get_firmware_fit_version");
+	if (aip->is_linux_available) {
+		ret = hlml_get_firmware_fit_version(aip, fw_fit_version, sizeof(fw_fit_version));
+		if (ret == HLML_SUCCESS)
+			HL_SMI_LOG_INFO("\tFirmware [FIT] Version\t\t: {}", fw_fit_version);
+		else
+			HL_SMI_LOG_ERR("FAIL in function: hlml_get_firmware_fit_version");
+	}
 
 	ret = hlml_get_firmware_spi_version(aip, fw_spi_version, sizeof(fw_spi_version));
 	if (ret == HLML_SUCCESS) {
